src/types.ts

```diff
@@ -0,0 +1,11 @@
+export enum EFormBuilderMode {
+    ADD_FIELD = "add-field",
+    EDIT_FIELD = "edit-field",
+    EDIT_PAGE = "edit-page",
+}
+
+export enum EFormBuilderView {
+    EDIT_FORM = "edit-form",
+    SCHEMA_PREVIEW = "schema-preview",
```

Comment:
think we wont be having this in the module. It'll be set up by the consumer

Reply:
this has not been rectified